### PR TITLE
Oral and written patterns

### PR DESCRIPTION
**Pull request by jswan5**
All plots have been moved from JavaScript to Plotly. Image links updated, LaTeX improved.

Defer to an education specialist regarding the wording.

### CODE REVIEW

**Comment by misterhay** [comment]
Great job rebuilding the interactives in Plotly. However since you re-used variable names like `fig` in the second interactive, once that is run it breaks the first interactive.

Would it be good to `update_layout` with something like `xaxis_range=[0,50]` to make it easier to see how the sliders are affecting the sequences?

I can start looking at wording next week.

**Comment by jswan5** [comment]
@misterhay I ran the plots after setting the range, but I found it had less clarity than allowing it to auto scale. That was especially true for the geometric scale, but that was to be expected.

I did add tick marks to the number line, I think that was a decent compromise? I also renamed the figure variables so that shouldn't be an issue anymore.

**Comment by misterhay** [comment]
The tick marks as a compromise work well, very nice.

I've made some commits to simplify the wording (and a minor code fix). It should be ready to merge, but perhaps @raniamahdi can have a look.

**Review by misterhay — APPROVED** [review]
Looks good. Thank you for your work on this
